File: js-pr/56/javascript/code-quality-performance/database_query_04.js
// {fact rule=code-quality-performance@v1.0 defects=1}
function getAllUsers() {
    let users = [];
    for (let i = 1; i <= 1000; i++) {
        let user = database.findUserById(i);
        users.push(user);
    }
    return users;
}
// {/fact}

File: python-pr/52/python/code-quality-performance/database_query_04.py
# {fact rule=code-quality-performance@v1.0 defects=1}
def get_all_users():
    users = []
    for i in range(1, 1001):
        user = database.find_user_by_id(i)
        users.append(user)
    return users
# {/fact}